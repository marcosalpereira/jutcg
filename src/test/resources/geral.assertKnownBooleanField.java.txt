package br.gov.esaf.sgc.business.impl;

public class Bean implements CampoAtuacaoBusiness {
    private Boolean field;

    public Boolean getField() {
        return field;
    }    
    
    public void resetField() {
        field = false;
    }

    public void activateField() {
        field = true;
    }
    
    public void nullifyField() {
        field = null;
    }
}