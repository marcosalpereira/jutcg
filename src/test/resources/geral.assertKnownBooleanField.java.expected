package br.gov.esaf.sgc.business.impl;

import static org.easymock.EasyMock.*;
import org.junit.*;
import java.util.*;
import static org.junit.Assert.*;

/**
 * Testes unitarios para a classe {@link Bean}.
 */
public class TestBean  {

/**
 * Classe sendo testada {@link Bean}.
 */
private Bean bean;

/**
 * Configuracoes iniciais.
 */
@Before
public void setUp() {
  bean = new Bean();
}

/**
 * Teste para o metodo {@link Bean#resetField()}.
 */
@Test
public void testResetField() {

// invocar metodo sendo testado
bean.resetField();

// verificacoes do resultado do metodo sendo testado
assertFalse(bean.getField());
}

/**
 * Teste para o metodo {@link Bean#activateField()}.
 */
@Test
public void testActivateField() {

// invocar metodo sendo testado
bean.activateField();

// verificacoes do resultado do metodo sendo testado
assertTrue(bean.getField());
}

/**
 * Teste para o metodo {@link Bean#nullifyField()}.
 */
@Test
public void testNullifyField() {

// invocar metodo sendo testado
bean.nullifyField();

// verificacoes do resultado do metodo sendo testado
assertNull(bean.getField());
}
}